src/components/button/Button.tsx

```diff
@@ -73,7 +74,7 @@ const createStyles = (type: ButtonType = ButtonType.PRIMARY) => (
           fontFamily: 'AvenirNextLTPro-Bold'
         },
         icon: {
-          marginLeft: 12
+          marginLeft: 6
```

Comment:
This will change the margin for every button - do we want that?

Reply:
^

Reply:
I reverted this, I thought 6px looked better across the board but for consistency with designs and web it should be 12px